src/pages/MyKiva/MyKivaPageContent.vue

```diff
@@ -668,4 +670,17 @@ export default {
 #recommended-loans :deep(.kv-carousel > div:first-child) {
 	@apply !tw-gap-2;
 }
+
+:deep(.bp-sidesheet-wrapper > div) {
```

Comment:
Why is this needed? What was the issue with created previously?

Reply:
Oh I see breakpoints. Can those be exported from the component?

Reply:
Yeah, works as expected... In fact we are passing those breakpoints to the component here:
https://github.com/kiva/ui/blob/31d23d323143bd40525360880439b9b2a282f860/src/pages/MyKiva/MyKivaPageContent.vue#L124

Just using the same for max widths.